XenAdmin/Dialogs/VmSnapshotDialog.cs

```diff
@@ -175,7 +175,7 @@ private void CheckpointInfoPictureBox_Click(object sender, EventArgs e)
         {
             string tt;
             if (Helpers.FeatureForbidden(_VM, Host.RestrictCheckpoint))
-                tt = string.Format(Messages.FEATURE_NOT_AVAILABLE_NEED_ENTERPRISE_OR_PLATINUM_PLURAL.Replace("\\n", "\n"), Messages.DISKMEMORY_SNAPSHOTS);
+                tt = Messages.FIELD_DISABLED;

```

Comment:
Can we add more information here like:
string.Format(Messages.FIELD_DISABLED.Replace("\\n", "\n"), Messages.DISKMEMORY_SNAPSHOTS);
So the detail reason can also go with the message.